runners/java-fn-execution/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
   compile library.java.guava
   compile library.java.findbugs_annotations
   compile project(path: ":beam-runners-core-construction-java", configuration: "shadow")
+  compileOnly project(path: ":beam-sdks-java-harness")
```

Comment:
Please use `provided`

Don't we need a corresponding change to the pom.xml?

Reply:
Done.

I can't find any documentation at all about the `provided` dependency scope in Gradle - every time I google it it sends me to `compileOnly`. This is a result of our `build_rules.gradle`?